src/coreclr/jit/morph.cpp

```diff
@@ -2291,36 +1910,25 @@ GenTree* Compiler::fgInsertCommaFormTemp(GenTree** ppTree, CORINFO_CLASS_HANDLE
     return new (this, GT_LCL_VAR) GenTreeLclVar(GT_LCL_VAR, subTree->TypeGet(), lclNum);
 }
 
-//------------------------------------------------------------------------
-// fgInitArgInfo: Construct the fgArgInfo for the call with the fgArgEntry for each arg
+// AddFinalArgsAndDetermineABIInfo:
```

Comment:
Are we dead set on the full template everywhere?
In this case the documentation would be something like "comp - the compiler" and "call - the call", which is not super useful and just adds noise. I have in many cases omitted documenting args when it would not clarify anything.

Reply:
It is our convention, yes.

I usually add them even if I think the args are self-descriptive, to at least distinguish those cases from the ones where the args might seem to be self-descriptive but aren't.

Reply:
Ok, let me include these as part of one of the follow-up changes to avoid rerunning CI for it.